pages/learn/index.js

```diff
@@ -70,54 +69,52 @@ class Learn extends React.Component {
               {this.state.filter === 'All'
                 ? 'Showing all courses'
                 : numOfFilteredSubjects === 0
-                  ? `Currenlty we dont have any subjects under ${this.state
-                      .filter}`
-                  : `Showing ${numOfFilteredSubjects} courses under ${this.state
-                      .filter}`}
+                  ? `Currenlty we dont have any subjects under ${
+                      this.state.filter
+                    }`
+                  : `Showing ${numOfFilteredSubjects} courses under ${
+                      this.state.filter
+                    }`}
             </Header>
             <Grid stackable columns={3}>
               {this.state.filteredSubjects.map(subject => (
                 <Grid.Column
                   style={{ paddingBottom: '4.5rem' }}
                   key={subject.id}
                 >
-                  <Link
-                    href={`/learn/subject?id=${subject.subjectId}`}
-                    as={subject.url}
+                  <Card
+                    href={subject.url}
```

Comment:
@pbteja1998 As we are using not using `<Link/>` component we loose client side navigation i.e the client sends a request to sever for this new page to load. Which is waste as we can reuse previously loaded client side bundle (where as with client side routing we only load js bundle data required to show new page). You can see this by opening chrome devtools -> Network Tab and compare above demo link and https://coderplex.org/learn 

---

So we don't want to remove `<Link/>` component and it is not what causing this issue. Semantic-ui-react's `<Card/>` is what preventing right-click so can you debug whats wrong with it. Thanks